change_column_type_to_lowcardinality/pipes/analytics_pages.pipe

```diff
@@ -0,0 +1,28 @@
+VERSION 1
```

Comment:
If I'm not wrong we're deprecating this, so shouldn't be used as example

Reply:
But are we using `v3` or `v2` in our examples?